runners/cli/src/test/kotlin/cli/CliTest.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.dokka
+
+import junit.framework.Assert.assertTrue
+import org.junit.Test
+import java.lang.IllegalStateException
+import kotlin.test.assertEquals
+
+class CliIntegrationTest {
+
+    @Test
+    fun testGlobalArgs() {
```

Comment:
Would be nice to correct the naming to represent expected behaviour, since new developers might not know what contracts `global` configs have. something like `should apply global settings to all source sets`